src/evtx_parser.rs

```diff
@@ -101,6 +104,18 @@ impl ParserSettings {
         };
         self
     }
+
+    #[cfg(not(feature = "multithreading"))]
+    pub fn num_threads(mut self, _num_threads: usize) -> Self {
```

Comment:
Add to docs that this does nothing when feature is disabled, maybe even print warning here :)